code/datums/uplink_item.dm

```diff
@@ -699,7 +699,7 @@ var/list/uplink_items = list()
 
 /datum/uplink_item/stealthy_weapons/dehy_carp
 	name = "Dehydrated Space Carp"
-	desc = "Just add water to make your very own hostile to everything space carp. It looks just like a plushie."
+	desc = "Just add water to make your very own hostile to everything space carp. It looks just like a plushie. \The first person to squeeze it will be registered as its owner, who it will not attack. If no owner is registered, it'll just attack everyone."
```

Comment:
it's also in error
this is defining `\The`
`\The` inserts "the" if the next embedded expression is an improper noun
newline is `\n`